src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -787,6 +787,32 @@ public void tupleTest() {
         chst.put(sr);
         chst.stop();
 
+        assertEquals(sr.size(), ClickHouseTestHelpers.countRows(chc, topic));
+    }
+    @Test
+    public void tupleTestWithDefault() {
+        Map<String, String> props = createProps();
+        ClickHouseHelperClient chc = createClient(props);
+        String topic = createTopicName("tuple-table-test-default");
+
+        ClickHouseTestHelpers.dropTable(chc, topic);
+        ClickHouseTestHelpers.createTable(chc, topic, "CREATE TABLE `%s` (" +
+                "`off16` Int16," +
+                "`string` String," +
+                "`insert_datetime` DateTime default now()," +
+                "`t` Tuple(" +
+                "    `off16` Nullable(Int16)," +
+                "    `string` Nullable(String) " +
```

Comment:
should we add a test case with a nested `Tuple`?

Reply:
Yep already in work

Reply:
Done